validator/accounts/wallet_edit.go

```diff
@@ -49,6 +49,8 @@ func EditWalletConfigurationCli(cliCtx *cli.Context) error {
 		if err := w.WriteKeymanagerConfigToDisk(cliCtx.Context, encodedCfg); err != nil {
 			return errors.Wrap(err, "could not write config to disk")
 		}
+	case keymanager.Web3Signer:
+		return errors.New("web3signer keymanager does not support editing configuration or persistent prysm wallet")
```

Comment:
I think you can remove this. It will be captured in the default with a defined error for not supported key manager. 

```suggestion
```

Reply:
Why was this resolved?